src/React/DOM/Props.purs

```diff
@@ -20,6 +19,7 @@ import React.SyntheticEvent
   , SyntheticUIEvent
   , SyntheticWheelEvent
   )
+import Web.HTML.HTMLElement (HTMLElement)
```

Comment:
Just a quick note. This is coming from `purescript-web-html`, but I didn't notice the dependency in the bower file. Is it being pulled in from another dependency?

On this note, I am unsure if we want this as a dependency since runtimes like react native, etc., would not require it. So far we have been attempting to keep dependencies at a minimum. That being said, I am not completely opposed to adding the dependency, but wanted to open it up for discussion.

Reply:
@ethul Well spotted; I think I forgot to push the change to the bower file. I had added `"purescript-web-html": "^2.2.0"` locally.

I suppose that we could have a local `HTMLElement` type here and not use the library version. It would save on the dependency, but on the other hand anyone using DOM refs would have to write a conversion function (i.e. `unsafeCoerce`), which might be annoying.

Reply:
Thanks for adding the dependency.